x-pack/plugins/apm/public/components/app/dependency_detail_overview/index.tsx

```diff
@@ -30,26 +30,29 @@ export function DependencyDetailOverview() {
 
   const apmRouter = useApmRouter();
 
-  useBreadcrumb([
-    {
-      title: i18n.translate(
-        'xpack.apm.dependencyDetailOverview.breadcrumbTitle',
-        { defaultMessage: 'Overview' }
-      ),
-      href: apmRouter.link('/dependencies/overview', {
-        query: {
-          dependencyName,
-          rangeFrom,
-          rangeTo,
-          refreshInterval,
-          refreshPaused,
-          environment,
-          kuery,
-          comparisonEnabled,
-        },
-      }),
-    },
-  ]);
+  useBreadcrumb(
+    [
+      {
+        title: i18n.translate(
+          'xpack.apm.dependencyDetailOverview.breadcrumbTitle',
+          { defaultMessage: 'Overview' }
+        ),
+        href: apmRouter.link('/dependencies/overview', {
+          query: {
+            dependencyName,
+            rangeFrom,
+            rangeTo,
+            refreshInterval,
+            refreshPaused,
+            environment,
+            kuery,
+            comparisonEnabled,
+          },
+        }),
+      },
+    ],
+    []
```

Comment:
This isn't correct - `apmRouter` and all the query parameters should be defined as dependencies. If you add `useBreadcrumb` to `.eslintrc.js` as suggested earlier you should see errors here too.